packages/lab/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -161,11 +76,7 @@ export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
 
```

Comment:
`rootProps["aria-valuemax"] = max;` maybe?

Reply:
also, would be great if it was consistent with the code in circular

Reply:
thanks, updated